HerPublicWebsite/Services/Cookies/CookieService.cs

```diff
@@ -90,6 +90,12 @@ public void SetCookie<T>(HttpResponse response, string cookieName, T cookie)
         response.Cookies.Append(
             cookieName,
             cookieString,
-            new CookieOptions {Secure = true, SameSite = SameSiteMode.Lax, MaxAge = TimeSpan.FromDays(Configuration.DefaultDaysUntilExpiry)});
+            new CookieOptions
+            {
+                Secure = true,
+                SameSite = SameSiteMode.Lax,
```

Comment:
Not strictly your change, but is there any reason this can't be `SameSiteMode.Strict` ?